domains/davitroncosop.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "davidtroncosop",
+    "email": "davidtroncosop@gmail.com"
+  },
+  "record": {
+    "A": "18.119.119.184"
```

cannot reach your website
```suggestion
    "A": ["18.119.119.184"]
```